requirements_gui.txt

```diff
@@ -1,7 +1,7 @@
-# fixes a memory leak. TODO: update to 4.7.3 when released
-git+git://github.com/enthought/enable@2c0a312f0b56a#egg=enable
+# TODO: move back to enthought (upstream) versions once they add python 3.7 support
+git+git://github.com/swift-nav/enable@swift-2019.04#egg=enable
+git+git://github.com/swift-nav/chaco@swift-2019.01#egg=chaco
```

Comment:
This is a random pull of chaco master, we should probably do the same Cython update exercise on 4.7.2 before we ship this.

Reply:
I added a tag `4.7.2+swift0` which updates the cython generated modules.

Reply:
```suggestion
git+git://github.com/swift-nav/chaco@4.7.2+swift0#egg=chaco
```